bitcoin/src/blockdata/constants.rs

```diff
@@ -84,7 +87,7 @@ fn bitcoin_genesis_tx() -> Transaction {
     });
 
     // Outputs
-    let script_bytes = hex!("04678afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef38c4f35504e51ec112de5c384df7ba0b8d578a4c702b6bf11d5f");
+    let script_bytes = PushBytesBuf::try_from(hex!("04678afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef38c4f35504e51ec112de5c384df7ba0b8d578a4c702b6bf11d5f")).expect("hex is valid length");
```

Comment:
This is **not** test code and this change turns compile-time error into runtime error. This was the main motivation behind making `hex_lit`.

Reply:
It turns a non-error into a non-error in code that will never need to change.

Is this really worth having an extra entry in our dep tree?

Would you consider folding `hex_lit` into `hex-conservative`?

Reply:
Is it worth the churn of doing PRs and stuff to change it?

> Would you consider folding `hex_lit` into `hex-conservative`?

Yes, I had that idea too. If that's something we want to do I'm fine with it. But honestly it annoys me that people are annoyed about the list of dependencies. (Well, I am annoyed too but only when it looks like a dependency is not transitively required for my project.)

Reply:
>But honestly it annoys me that people are annoyed about the list of dependencies. (

Me too, but here we are after 17 years of Rust and the only builtin tool for managing dependencies is `cargo tree`, which doesn't even show authorship information. I've tried multiple tools like `cargo-outdated` but these themselves always have 200+ dependencies, tend to break, and have overlapping functionality such that no single one quite does everything we want.

>Yes, I had that idea too. If that's something we want to do I'm fine with it

Ok, thank you!